openslides_backend/permissions/base_classes.py

```diff
@@ -24,3 +26,11 @@ def get_verbose_type(self) -> str:
 
     def get_base_model(self) -> str:
         return "meeting"
+
+
+class BasePermissionsContainer:
+    @classmethod
+    def parse(cls, permission: str) -> Permission:
```

Comment:
Where is it used?

Reply:
Also the class itself is not used, right?

Reply:
These are artifacts from rebasing, I removed them in master and this branch is not rebased yet

Reply:
Okay, this will be removed then.